### PR TITLE
common: code refactoring

### PR DESCRIPTION
**Pull request by hermet**
introduced mathDeg2Rad() and mathRad2Deg() for a common implementation.

### CODE REVIEW

File: src/common/tvgMath.h [hunk 1]
```diff
@@ -44,6 +44,18 @@ bool mathIdentity(const Matrix* m);
 void mathMultiply(Point* pt, const Matrix* transform);
 
 
+static inline float mathDeg2Rad(float degree)
+{
+     return degree * MATH_PI / 180.0f;
```
Comment (JSUYA) [inline src/common/tvgMath.h:49]: how about re-define `MATH_PI / 180.0f` ?
```
#define MATH_PI3 0.01745329238474369049f
```

File: src/common/tvgMath.h [hunk 2]
```diff
@@ -44,6 +44,18 @@ bool mathIdentity(const Matrix* m);
...
+}
+
+
+static inline float mathRad2Deg(float radian)
+{
+    return radian * 180.0f / MATH_PI;
```
Comment (JSUYA) [inline src/common/tvgMath.h:55]: how about re-define `180.0f / MATH_PI` ?
```
#define MATH_PI4  57.2957763671875f
```

File: src/common/tvgMath.h (hunk 1 above)
Comment (hermet) [reply src/common/tvgMath.h:49]: I don't see any beneficial points. Please think about these points:
1. better performance?
2. less & neat code?
3. enhance the readibility?

Comment (JSUYA) [reply src/common/tvgMath.h]: 1. A division operation is performed every time. The amount of computation can be reduced.
2.3.4. You have already defined `MATH_PI *0.5f` as `MATH_PI2`. If you use an appropriate name, I think it is better.

Comment (hermet) [reply src/common/tvgMath.h]: - Compiler will do pre-computation. No benefits practically.
- MATH_PI2 has followed the M_PI2, as in the math definition. This prevents the repetitive code from the usage.
- Current logic is already commonized and clear to read. Please suggest me the proper name if you'd like to argue this. 
- In the end, those definition introduction will just increase the LOC.

**Review by SergeyLebedkin — APPROVED** [review]

**Review by tinyjin — APPROVED** [review]

**Pull request merged**